backends/gstreamer/src/player.rs

```diff
@@ -123,6 +123,16 @@ impl PlayerInner {
         self.input_size = size;
     }
 
+    pub fn set_seekable(&mut self, seekable: Seekable) {
+        if let Some(ref appsrc) = self.appsrc {
+            appsrc.set_stream_type(match seekable {
+                Seekable::NonSeekable => gst_app::AppStreamType::Stream,
+                Seekable::Seekable => gst_app::AppStreamType::Seekable,
+                Seekable::SeekableFast => gst_app::AppStreamType::RandomAccess,
```

Comment:
I'm not sure if SeekableFast makes sense in the Servo's context, since it depends in the ``FetchReponseListener``'s methods.